.github/workflows/setup-project.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
         sudo apt-get update && sudo apt-get install --only-upgrade python3       
         INSTALL_URL="https://raw.githubusercontent.com/sixfeetup/scaf/$GITHUB_REF_NAME/install.sh"
         export SCAF_SCRIPT_BRANCH=$GITHUB_REF_NAME
-        git config --global init.defaultBranch main
+        git config --global init.defaultBranch dev
```

Comment:
@rochecompaan not sure if we need to change any scaf workflow to run against dev branch. I think semantic release should also stay as it is and run on main. Do you know any of them that needs an update?

Reply:
@sedatbasar that's correct, semantic release should only run once dev is merged into main.